src/content/ru/updates/2018/12/class-fields.md

```diff
@@ -0,0 +1,219 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: Several proposals expand the existing JavaScript class syntax with new functionality. This article explains the new public class fields syntax in V8 v7.2 and Chrome 72, as well as the upcoming private class fields syntax.
+
+{% include "web/_shared/machine-translation-start.html" %}
```

Comment:
We can remove this, as it's not a machine translation :)

Reply:
Oh, good! I'll append my credit later :) Thanks for that opportunity!

Reply:
Or was this a machine translation with just a few tweaks? In that case, we should keep the line. @PaulKinlan

Reply:
No, this is my own interpretation of the material :)